### PR TITLE
New Tutorial: OMERO-suite for Galaxy

### PR DESCRIPTION
**Pull request by rmassei**
**New Imaging Tutorial on the OMERO-suite for Galaxy**

**Summary:**
This PR introduces a new tutorial for using the different tools of the OMERO-suite. This includes image and metadata import, data fetching, filtering and ROI upload. The tutorial the describes the use of the single tools, plus it give an example of a pipeline to upload a single image and link metadata or ROIs.

**Help Needed**:
First time writing a Galaxy tutorial, it would be great to have a general feedback :)

### CODE REVIEW

**Comment by bgruening** [comment]
@rmassei let us please know if you need any help here!

File: topics/imaging/tutorials/omero-suite/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+>   - Connect the output of {% icon tool %} **7: OMERO Image Import** to the {% icon tool %} **8: Convert** input
+>2. Add {% icon tool %} **Replace text**  from the list of tools:
+>   - Connect the output of {% icon tool %} **8: Convert** to the {% icon tool %} **Replace text** input 
+>3. Add {% icon tool %} **Split File**  from the list of tools:
+>   - Connect the output of {% icon tool %} **9: Replace text** to the {% icon tool %} **10: Split File** 
+>4. Add {% icon tool %} **Parse parameter value**  from the list of tools:
+>   - Connect the output of {% icon tool %} **10: Split File**  to the {% icon tool %} **11: Parse parameter value**
+{: .hands_on}
+
+This sequence of tools dynamically parses the single image ID which are use them for input of the metadata and
```
Comment (bgruening) [inline topics/imaging/tutorials/omero-suite/tutorial.md:338]: Can you please look at this sentence, I was not able to understand it properly to fix it.

File: topics/imaging/tutorials/omero-suite/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+
+- Allan, C., Burel, JM., Moore, J. et al. OMERO: flexible, model-driven data management for experimental biology. 
+Nat Methods 9, 245–253 (2012). https://doi.org/10.1038/nmeth.1896
+- Wilkinson, M., Dumontier, M., Aalbersberg, I. et al. The FAIR Guiding Principles for Scientific Data Management and
+stewardship. Sci Data 3, 160018 (2016). https://doi.org/10.1038/sdata.2016.18
+
+## Acknowledgement
+
+We acknowledge the support of [**NFDI4BIOIMAGE**](https://nfdi4bioimage.de/home/), 
+funded by the German Research Foundation (DFG) within the framework of the NFDI – project numbers: 501864659.
```
Comment (bgruening) [inline topics/imaging/tutorials/omero-suite/tutorial.md:392]: This is not needed, as you can and have annotated the tutorial with the proper funding and the references can be references via a citations file.

File: CONTRIBUTORS.yaml [hunk 3]
```diff
@@ -2710,3 +2719,11 @@ tbrown91:
...
     orcid: 0000-0001-8293-4816
     joined: 2024-08
+
+rmassei:
+    name: Riccardo Massei
+    email: riccardo.massei@ufz.de
+    orcid: 0000-0003-2104-9519
+    joined: 2024-11
+    affiliations:
+      - ufz
```
Comment (bgruening) [inline CONTRIBUTORS.yaml:2729]: I think you also need to add the UFZ

Comment (rmassei) [reply CONTRIBUTORS.yaml]: ok, just left nfdi4bioimage, I also realized that my name was 2 times in the contributors list

**Comment by bgruening** [comment]
> topics/imaging/tutorials/omero-suite/tutorial.md:114: Potential broken box. A hands_on was opened on 82, but not closed on line 114
topics/imaging/tutorials/omero-suite/tutorial.md:362: Potential broken box. A hands_on was opened on 343, but not closed on line 362

File: topics/imaging/tutorials/omero-suite/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+# Upload images and metadata
+
+In this section, we will focus on how uploading images and metadata into a user-defined OMERO instance.
+This is done by using the **OMERO Image Import** and **OMERO Metadata Import** tools.
+
+## Step 1: Image Upload
+
+> <hands-on-title>Upload images into OMERO</hands-on-title>
+>
+> 1. {% tool [OMERO Image Import](https://toolshed.g2.bx.psu.edu/view/ufz/omero_import/5.18.0+galaxy3) %} with the following recommended parameters:
```
Comment (bgruening) [inline topics/imaging/tutorials/omero-suite/tutorial.md:126]: Those are tool IDs, they need to look like `toolshed.g2.bx.psu.edu/repos/bgruening/black_forest_labs_flux/black_forest_labs_flux/2024+galaxy2`

You can find this link in the tool menue
grafik

Comment (bgruening) [reply topics/imaging/tutorials/omero-suite/tutorial.md:126]: This would need to be changed in all tools.

**Comment by rmassei** [comment]
Thanks for supporting along all the process :D

**Pull request merged**